# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1504911755 25200
#      Fri Sep 08 16:02:35 2017 -0700
# Node ID fbdefffeaa2134c447dcaec8f6988e41efff148b
# Parent  1cb94f46466fa2627f04aff02a2c98e19c20cd96
build: ensure liboctave/util/*.f are built with right integer size (bug #51966)

* liboctave/util/module.mk (%canon_reldir%_libutil_la_FFLAGS): Define and
include $(F77_INTEGER_8_FLAG).

diff --git a/liboctave/util/module.mk b/liboctave/util/module.mk
--- a/liboctave/util/module.mk
+++ b/liboctave/util/module.mk
@@ -111,9 +111,11 @@ noinst_LTLIBRARIES += %reldir%/libutil.l
   $(CURL_CPPFLAGS) \
   $(PCRE_CPPFLAGS) \
   $(SPARSE_XCPPFLAGS)
 
 %canon_reldir%_libutil_la_CFLAGS = $(liboctave_liboctave_la_CFLAGS)
 
 %canon_reldir%_libutil_la_CXXFLAGS = $(liboctave_liboctave_la_CXXFLAGS)
 
+%canon_reldir%_libutil_la_FFLAGS = $(F77_INTEGER_8_FLAG)
+
 liboctave_liboctave_la_LIBADD += %reldir%/libutil.la
